src/server/JasmineGraphServer.cpp

```diff
@@ -240,6 +241,25 @@ void JasmineGraphServer::start_workers() {
 
 }
 
+void JasmineGraphServer::waitForAcknowledgement(int numberOfWorkers) {
+    auto begin = chrono::high_resolution_clock::now();
+    int timeDifference = 0;
+    while (timeDifference < 30000) {
```

Comment:
Lets define this value `30000` as a constant in the `Conts.cpp` file in the `util` and then refer it here.

Reply:
I'll move that to a constant

Reply:
Ok